RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -65,7 +74,9 @@
     ptMin = 10,
     deltaPhiRegion = 0.20,
     deltaEtaRegion = 0.20,
-    JetSrc = "jetsForCoreTracking",
+#    JetSrc = "jetsForCoreTracking",
+ #   JetSrc = "egammasForCoreTracking",
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
removed